dev-docs/bidders/sortable.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: bidder
+title: Sortable
+description: Prebid Sortable Bidder Adaptor
+top_nav_section: dev_docs
+nav_section: reference
+biddercode: sortable
+biddercode_longer_than_12: false
+hide: true
+prebid_1_0_supported : true
+media_types: banner
+gdpr_supported: true
+---
+
+**Table of Contents**
+
+- [Bid params](#sortable-bid-params)
+- [Banner Ads](#sortable-banner)
+- [Configuration](#sortable-configuration)
+
+<a name="sortable-bid-params" />
+
+### Bid params
+
+{: .table .table-bordered .table-striped }
+| Name | Scope | Description | Example |
+|------+-------+-------------+---------|
+| `tagId` | required | The tag ID from Sortable. | `"test-pb-leaderboard"` |
+| `siteId` | required | The site ID from Sortable. To blanket set the site ID, [see here](#sortable-configuration). Please reach out to your Sortable Account Manager for more details. | `1` or `"example.com"` |
+| `keywords` | optional | Publisher-defined key-value string pairs | `{ "key1": "val1", "key2": "val2" }` |
+
+<a name="sortable-banner" />
+
+### Banner Ads
+
+Sortable supports the banner features described in:
+
+- [The `adUnit` banner documentation]({{site.baseurl}}/dev-docs/adunit-reference.html#adUnit-banner-example)
+- [Getting Started for Developers]({{site.baseurl}}/dev-docs/getting-started.html)
+
+
+<a name="sortable-configuration" />
+
+### Configuration
+
+If the Sortable site ID is set globally, `siteId` is not required as a bid param. How to do so:
+
+```javascript
+$$PREBID_GLOBAL$$.setConfig({
```

Comment:
I'd suggest the example be `pbjs.setConfig` instead of `$$PREBID_GLOBAL$$`

Also - would suggest defining a top-level `sortable` object in setConfig rather than `sortableId`. All of our adapter-specific items would go there. e.g.
```
pbjs.setConfig({
   sortable: {
      id: "example.com"
   }
)};
```